lib/net/ldap.rb

```diff
@@ -540,6 +540,9 @@ def initialize(args = {})
     @base = args[:base] || DefaultTreebase
     @force_no_page = args[:force_no_page] || DefaultForceNoPage
     @encryption = args[:encryption] # may be nil
+    if !@encryption.nil? and !@encryption.is_a?(Hash)
```

Comment:
These operators still hurt my brain, if we replace the `and` operators in the this codebase to `&&` all the tests fail.

http://devblog.avdi.org/2014/08/26/how-to-use-rubys-english-andor-operators-without-going-nuts/